engines/director/detection_tables.h

```diff
@@ -4722,6 +4733,10 @@ static const DirectorGameDescription gameDescriptions[] = {
 	// Win/Mac versions are D3
 	PIPGAME1_l("lzone", "", "L-ZONE", "07c3095bd36e5aed00ba12f17f761070", 60096, Common::JA_JPN, 400),
 
+	MACGAME1_l("lulu", "", "Das Buch von Lulu PowerMac", "d:ab6ba48edb4a93ca034c63905f615865", 789962, Common::DE_DEU, 404),
```

Comment:
Same as above, should use the resfork and `r:` before the checksum

Reply:
Fixed and force-pushed.